designs/UI/login.html

```diff
@@ -0,0 +1,80 @@
+                <div class="row"></div>
+        </nav>
+        
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-6 col-sm-4"></div>
+            <div class="col-xs-6 col-sm-4">
+            <div class="text-primary">
+                 <h3 class="text-center font-weight-bold">Log in Form</h3>
+             </div>
+            </div>
+            <div class="col-xs-6 col-sm-4"></div>
+        </div>
+    </div>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-6 col-sm-4"></div>
+            <div class="col-xs-6 col-sm-4">
+            <div class="card border-primary text-primary mb-3">
+                <div class="card-header font-weight-bold">Members Log in</div>
+                <div class="card-body">
+                    <p class="text-center">Please enter your credentials to login</p>
+                    <form class="">
+                    
+                        <div class="form-group">
+                                <div class="inner-addon left-addon">
+                                    <i class="fa fa-user text-primary"></i>
+                                    <input type="text" class="form-control" id="username" placeholder="Username or email">
+                                </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="inner-addon left-addon">
+                                <i class="fa fa-lock text-primary"></i>
+                                <input type="password" class="form-control" id="lastname" placeholder="Password">
+                            </div>
+                        </div>
+                        <div class="form-group">
+                                <div class="checkbox text-center">
+                                    <a class="medium-text"><input type="checkbox"> Remember me</a>
+                                </div>
+                        </div>
+                        <div class="form-group text-center">
+                                <button type="submit" class="button button4 btn-primary">Sign in >></button>
+                                <p class="text-center medium-text">Don't have an account
+                                    <a class="text-success" href="register.html">REGISTER HERE</a></p>
+                            </div>
+                        </div>
+                        
+                    </form>
+                </div>
+                </div>
+            </div>
+            <div class="col-xs-6 col-sm-4"></div>
+        </div>
+    </div>
+    
+    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://code.jquery.com/jquery.js"></script>
+    <!-- Include all compiled plugins (below), or include individual files
+     as needed -->
+    <script src="js/bootstrap.min.js"></script>   
+</body>
+</html>
```

Comment:
Add an extra line at the end of your file

Reply:
ok